x-pack/plugins/apm/public/components/app/Main/AccessControl/UpgradeAPMServerNotification.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import React from 'react';
+
+const UPGRADE_APM_SERVER_URL =
+  'https://www.elastic.co/guide/en/apm/server/current/upgrading.html';
```

Comment:
We have all the docs links together in https://github.com/elastic/kibana/tree/master/x-pack/plugins/apm/public/utils/documentation.

Can you move this link over to a `server.ts` file? We should probably also link to the stack version like we do with the other docs links: https://github.com/jasonrhodes/kibana/blob/39cda0facd4442aa4403ce95ae610d17614b8743/x-pack/plugins/apm/public/utils/documentation/apm-get-started.ts#L9